src/components/PhotoPage.js

```diff
@@ -112,30 +120,31 @@ class PhotoPage extends Component {
   }
 
   componentDidMount() {
-    this.loadImage();
-
     window.gtag('event', 'page_view', {
       'event_category': 'view',
       'event_label': 'PhotoPage'
     });
   }
 
+  componentDidUpdate = (nextProps) => {
```

Comment:
nextProps should be renamed to prevProps
https://reactjs.org/docs/react-component.html#componentdidupdate